examples/groupchat/main.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+""" A group chat where user can talk any time implemented by agentscope. """
+from groupchat_utils import (
+    select_next_one,
+    filter_names,
+)
+
+import agentscope
+from agentscope.agents import UserAgent
+from agentscope.message import Msg
+from agentscope.msghub import msghub
+from agentscope.utils.common import timer
+
+USER_TIME_TO_SPEAK = 10
```

Comment:
It's suggested to add some comments to help developers to better understand the code.